Translation corrections (#129)
07th-mod/higurashi-console-arcs: Update/some_011.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300668", 256, TRUE);
	OutputLine(NULL, "『あったり前でしょ！",
			NULL, "\"Damn straight she did!", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300669", 256, TRUE);
	OutputLine(NULL, "　初めて会った時に、私たちが気づかなかったとでも思ってたのぉ？",

> 			NULL, " Since we first met! You thought we hadn't notice?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Since we first met! You thought we hadn't noticed?", Line_WaitForInput);
```

//r知ってたじゃないか。私が藤堂くんのことを好きだって、…初めて会ったその時から！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "知ってたじゃないか。私が藤堂くんのことを好きだって、…初めて会ったその時から！",

> 			NULL, "She knew, didn't she? She knew ever since we first met that... that I liked Toudou-kun!", Line_Normal);

Reviewer: ```suggestion
			NULL, "She knew, didn't she? She knew ever since we first met... that I liked Toudou-kun!", Line_Normal);
```

	ClearMessage();

	DrawScene("background/aka1", 1000 );

//rだったらなんで、二人とも教えてくれない？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だったらなんで、二人とも教えてくれない？",
			NULL, "So why didn't the two of them tell me?", Line_Normal);
	ClearMessage();

//r私だけ仲間はずれにして、二人して私を笑っていたの？！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私だけ仲間はずれにして、二人して私を笑っていたの？！",
			NULL, "Were they laughing at me behind my back?!", Line_Normal);
	ClearMessage();

//r高校を卒業すれば別の学校に行って、藤堂くんと離れ離れになることを知ってて、二人とも私のことをおだてて、いい気にさせて、…その裏では馬鹿にしていたのかッ？！
	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "高校を卒業すれば別の学校に行って、藤堂くんと離れ離れになることを知ってて、二人とも私のことをおだてて、いい気にさせて、…その裏では馬鹿にしていたのかッ？！",

> 				NULL, "<size=-2>When I graduate from high school, I'll be going to a different school and I'll become separated from Toudou-kun. They both knew that, flattered me, cheered me up... all while secretly making fun of me?!", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>When I graduate from high school, I'll be going to a different school and I'll become separated from Toudou-kun. They both knew that, flattered me, cheered me up... all while secretly mocking me?!", Line_Normal);
```

	} else {
		OutputLine(NULL, "高校を卒業すれば別の学校に行って、藤堂くんと離れ離れになることを知ってて、二人とも私のことをおだてて、いい気にさせて、…その裏では馬鹿にしていたのかッ？！",

> 				NULL, "When I graduate from high school, I'll be going to a different school and I'll become separated from Toudou-kun. They both knew that, flattered me, cheered me up... all while secretly making fun of me?!"", Line_Normal);

Reviewer: ```suggestion
				NULL, "When I graduate from high school, I'll be going to a different school and I'll become separated from Toudou-kun. They both knew that, flattered me, cheered me up... all while secretly mocking me?!"", Line_Normal);
```

//rはっ、と我に返って気がつくと。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "はっ、と我に返って気がつくと。",

> 			NULL, "I came back to my senses and noticed it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "When I came back to my senses, I noticed.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "少し大声を張り上げすぎたせいで、店内のお客さんが何事かとこちらを振り向く。その居心地の悪さに耐えられず、私は席を立って店を出、…ようとして、足元がふらつきその場にへなへなと崩れ落ちた。",
				NULL, "<size=-2>The customers in the store began turning to look at source of the loud noises. I couldn't stand the uncomfortable feeling, so I stood up and begin to make my way out of the restaurant... At least, that was the plan, but my feet began to wobble and I crumbled to the ground.", Line_Normal);
	} else {
		OutputLine(NULL, "少し大声を張り上げすぎたせいで、店内のお客さんが何事かとこちらを振り向く。その居心地の悪さに耐えられず、私は席を立って店を出、…ようとして、足元がふらつきその場にへなへなと崩れ落ちた。",

> 				NULL, "he customers in the store began turning to look at source of the loud noises. I couldn't stand the uncomfortable feeling, so I stood up and begin to make my way out of the restaurant... At least, that was the plan, but my feet began to wobble and I crumbled to the ground.", Line_Normal);

Reviewer: ```suggestion
				NULL, "The customers in the store began turning to look at source of the loud noises. I couldn't stand the uncomfortable feeling, so I stood up and begin to make my way out of the restaurant... At least, that was the plan, but my feet began to wobble and I crumbled to the ground.", Line_Normal);
```

//r情けなくて、恥ずかしくって、…私は、嗚咽とともに涙を流した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "情けなくて、恥ずかしくって、…私は、嗚咽とともに涙を流した。",
			NULL, "I felt miserable and embarrassed... Tears started flowing out, and I began to sob.", Line_Normal);
	ClearMessage();

//rわからない。今日は朝から体調も良くて、気分も最高潮だったのに。急に全身が自分のものでなくなったように、言うことを聞いてくれなくなる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "わからない。今日は朝から体調も良くて、気分も最高潮だったのに。急に全身が自分のものでなくなったように、言うことを聞いてくれなくなる。",
			NULL, "I didn't understand. I felt healthy this morning and I was in a great mood. Yet suddenly, my body was no longer listening to me, as if it wasn't mine anymore.", Line_Normal);
	ClearMessage();

//r…なんでなの？　なんで私なの？　なんでこの日なの？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…なんでなの？　なんで私なの？　なんでこの日なの？",
			NULL, "...Why is this happening? Why me? Why today?", Line_Normal);
	ClearMessage();

//r今日は私にとって、とっても、…とっても、何よりも大事な日なのに…ッ！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "今日は私にとって、とっても、…とっても、何よりも大事な日なのに…ッ！",

> 			NULL, "Even though this is the most important day of my life yet...!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even though this is the most important day of my life...!", Line_Normal);
```
